### PR TITLE
Forward fix Fix batch norm partitioning with Conv3d (#13696)

### PR DESCRIPTION
**Pull request by mergennachin**
Summary:
Forward pyre fix

https://www.internalfb.com/diff/D81069236
https://github.com/pytorch/executorch/pull/13696

bypass-github-export-checks
bypass-github-pytorch-ci-checks
bypass-github-executorch-ci-checks

Reviewed By: digantdesai

Differential Revision: D82118651

### CODE REVIEW

**Review by digantdesai — APPROVED** [review]
Review automatically exported from Phabricator review in Meta.
